src/EncompassRest/Contacts/BorrowerContacts.cs

```diff
@@ -0,0 +1,161 @@
+﻿using EncompassRest.Utilities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace EncompassRest.Contacts
+{
+    public sealed class BorrowerContacts
+    {
+        private const string s_apiPath = "encompass/v1/BorrowerContacts";
+
+        #region Public Properties
+        public EncompassRestClient Client { get; }
+        #endregion
+
+        internal BorrowerContacts(EncompassRestClient client)
+        {
+            Client = client;
+        }
+
```

Comment:
Add `GetContacts` methods, it's not documented but is supported in the APIs.

Reply:
is it?

Reply:
ok, it is supported, but not set up for large data yet.  we have 85,000 borrower contacts, and the request times out.

Reply:
Maybe hold off then for now as I'm sure there'll be a cursor added later.